da/da.go

```diff
@@ -33,12 +34,12 @@ type BaseResult struct {
 	DAHeight uint64
 }
 
-// ResultSubmitBlock contains information returned from DA layer after block submission.
+// ResultSubmitBlock contains information returned from DA layer after block header/data submission.
 type ResultSubmitBlock struct {
```

Comment:
header and data are part of block. `ResultSubmitBytes` is too generic and does not imply block relation. note that, we still can the two DALC APIs, `SubmitBlockHeader` and `SubmitBlockData`. i feel it will introduce more confusion by being generic?

Reply:
ResultSubmitBob could be good. Block confused me when reading the code as it implies something.

Reply:
i would like to address this later, as i cannot convince myself why this has to be re-named. should be ok?